diff --git a/node_modules/expo-dev-launcher/expo-dev-launcher.podspec b/node_modules/expo-dev-launcher/expo-dev-launcher.podspec
index d80a559..1e8f2ba 100644
--- a/node_modules/expo-dev-launcher/expo-dev-launcher.podspec
+++ b/node_modules/expo-dev-launcher/expo-dev-launcher.podspec
@@ -108,7 +108,9 @@ Pod::Spec.new do |s|
   s.dependency "EXUpdatesInterface"
   s.dependency "expo-dev-menu"
   s.dependency "ExpoModulesCore"
-  s.dependency "ReactAppDependencyProvider"
+  if new_arch_enabled
+    s.dependency "ReactAppDependencyProvider"
+  end
   add_dependency(s, "React-jsinspector", :framework_name => 'jsinspector_modern')
 
   unless defined?(install_modules_dependencies)
