diff --git a/node_modules/expo-dev-menu/expo-dev-menu.podspec b/node_modules/expo-dev-menu/expo-dev-menu.podspec
index 5e104cf..4b7a74e 100644
--- a/node_modules/expo-dev-menu/expo-dev-menu.podspec
+++ b/node_modules/expo-dev-menu/expo-dev-menu.podspec
@@ -115,8 +115,6 @@ Pod::Spec.new do |s|
   s.test_spec 'UITests' do |test_spec|
     test_spec.requires_app_host = true
     test_spec.source_files = 'ios/UITests/**/*'
-    test_spec.dependency 'React-CoreModules'
-    test_spec.dependency 'ReactAppDependencyProvider'
     test_spec.dependency 'React'
     test_spec.platforms = {
       :ios => '15.1'
